src/index.html

```diff
@@ -8,6 +8,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star__container">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="star__container">
+        <div class="stars__star"></div>
+      </div>
```

Comment:
why do we need those star__containers?

Reply:
because of positioning of bg-image. Image of star doesn't look like in Figma Example. In example it's width = 16px and height = 16px, but in default size width = 13px and height = 12px

Reply:
You can use `background-position` for it, remove these containers